app/src/modules/settings/routes/extensions/extensions.vue

```diff
@@ -56,10 +84,17 @@ fetchExtensions();
 			<extensions-info-sidebar-detail />
 		</template>
 
+		<div v-if="needsReload" class="page-container">
+			<v-notice type="warning">
+				{{ t('extension_reload_required_copy') }}&nbsp;
+				<a :href="currentPageLink()" @click="removeGuard">{{ t('extension_reload_now') }}</a>
```

Comment:
See comment above regarding `removeGuard`

```suggestion
				<a :href="currentPageLink">{{ t('extension_reload_now') }}</a>
```